WorkloadManagementSystem/Agent/PilotStatusAgent.py

```diff
@@ -505,3 +515,32 @@ def __addPilotsAccountingReport( self, pilotsData ):
       if not retVal[ 'OK' ]:
         return retVal
     return S_OK()
+
+  def _killPilots( self, acc ):
+    for i in sorted(acc.keys()):
+      result = self.diracadmin.getPilotInfo( i )
+      if result['OK'] and result['Value'].has_key(i) and result['Value'][i].has_key('Status'):
+        ret = self.diracadmin.killPilot( str(i) )
+        if ret['OK']:
+          self.log.info("Sucsessfully deleted: %s (Status : %s)" % (i, result['Value'][i]['Status'] ) )
+        else:
+          self.log.error("Filed to delete %s : %s"  % ( i, ret['Message']))
+      else:
+        self.log.error("Failed to get info. of %s : %s" % ( i, str(result)))
+
+  def _CheckJobLastUpdateTime( self, joblist , StalledDays ):
```

Comment:
The DIRAC naming convention stipulates that method names must start with lower case character if even preceded by an underscore